src/unlimited.cpp

```diff
@@ -1153,7 +1153,16 @@ bool IsChainNearlySyncd()
     LOCK(cs_ischainnearlysyncd);
     return fIsChainNearlySyncd;
 }
-
+uint64_t LargestBlockSeen(uint64_t nBlockSize)
+{
+     // return the largest block size that we have seen since startup
+     static uint64_t nLargestBlockSeen = BLOCKSTREAM_CORE_MAX_BLOCK_SIZE;
```

Comment:
gmaxwell is correct (but wrong too). Terms like "BLOCKSTREAM_CORE..." inside the codebase are just inserting political bullshit into your code. It should obviously be changed to something more sensible. Leaving it there just makes BU look silly.

He is wrong though in claiming in **no way** a legal threat was made. Using terms like trademark and defamatory imply a legal basis for pursuing action (especially when done by an officer of a company), and implied legal action is a form of legal threat ("we could sue you" is not "we will sue you if", but still a threat). 

Anyway the variable name should be changed, regardless of any trademark issues.